extensions/azdata/src/common/logger.ts

```diff
@@ -14,6 +14,7 @@ export class Log {
 
 	log(msg: string): void {
 		this._output.appendLine(msg);
+		console.log(msg);
```

Comment:
Remove this - or at least make it optional somehow. I don't want our normal console logs being cluttered with a ton of random stuff.

Reply:
Done.